xds/internal/client/client.go

```diff
@@ -341,46 +315,40 @@ type Client struct {
 	lrsClients map[string]*lrsClient
 }
 
-// New returns a new xdsClient configured with opts.
-func New(opts Options) (*Client, error) {
+// newWithConfig returns a new xdsClient with the given config.
+func newWithConfig(config *bootstrap.Config, watchExpiryTimeout time.Duration) (*clientImpl, error) {
```

Comment:
Would it better to keep the `Options` struct (unexport it maybe)? Tomorrow if we need to pass something else to this client, it might be easier with a struct.

Reply:
I was thinking the things we need would all come from `bootstrap.Config`. The expiry timeout is only added for testing (we could also make it a global variable that tests override, but I like this a little better).